src/searchee.ts

```diff
@@ -19,6 +20,39 @@ export interface Searchee {
 	length: number;
 }
 
+function getFilesFromPath(dirPath): File[] {
+	if (fs.statSync(dirPath).isDirectory()) {
+		var files: string[] = getFilePathsFromPath(dirPath, []);
+	} else {
+		var files: string[] = [dirPath];
+	}
```

Comment:
can we absorb this conditional inside the helper `getFilePathsFromPath`?

Reply:
I could try to absorb it, but it might not look exactly like this. It has to be properly recursive since there's no way to know how far the torrent directory tree goes. And to be honest, I'd rather not refactor it. It's a little ugly and inelegant right now but I had issues when I initially tried to do it this way. Maybe my recursion skills have fallen off...